lib/service/storage/elasticsearch.ts

```diff
@@ -1966,12 +2099,19 @@ class ElasticSearch extends Service {
    *
    * @returns {Promise.<String[]>}
    */
-  async deleteIndexes(indexes = []) {
+  async deleteIndexes(indexes = []): Promise<string[]> {
     if (indexes.length === 0) {
       return Bluebird.resolve([]);
     }
-    const deleted = new Set();
-
+    const deleted = new Set<string>();
```

Comment:
I think it could be a simple array here and it will avoid the `Array.from` at the end

Reply:
done

Reply:
Undo : That can create a fail in an unit test because it return many time the same name in the deleted array.